portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,44 +24,47 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private static final String[] internalTags = {"#general", "#wellbeing", "#music", "#games"};
+  private final String[] INTERNAL_TAGS = {"#general", "#wellbeing", "#music", "#games"};
+  private Map<String, LinkedList<BlogMessage>> map;
+
+  // Returns whether a message's tag is supported internally.
+  private boolean tagIsSupported(BlogMessage message) {
+    boolean isTagSupported = false;
+    for (int i = 0; i < INTERNAL_TAGS.length; i++){
+      if (message.getTag().equals(INTERNAL_TAGS[i])){
+        isTagSupported = true;
+      }
+    }
+    return isTagSupported;
+  }
 
   // Takes and puts a list of BlogMessage type in map.
   // Each BlogMessage type should contain tag that matches tags in the internal System. 
-  public static void putInMap(List<BlogMessage> messages, Map<String, LinkedList<BlogMessage>> map) {
+  public void putInMap(List<BlogMessage> messages) {
     for (BlogMessage message : messages) {
-      boolean isTagSupported = false;
-      for (int i = 0; i < internalTags.length; i++){
-        if (message.getTag().equals(internalTags[i])){
-          isTagSupported = true;
-          if (map.containsKey(message.getTag())){
-            map.get(message.getTag()).addLast(message);
-          } else {
-            // No such mapping with specified tag,
-            // Create first mapping with specified tag.
-            LinkedList<BlogMessage> firstValue = new LinkedList<BlogMessage>();
-            firstValue.addFirst(message); 
-            map.put(message.getTag(), firstValue);
-          }
-        } 
-      }
-      if (!isTagSupported) {
-        System.out.println("Tag not supported. Tag: " + message.getTag());
-        System.out.println("For BlogMessage@ " + message);
+      if (tagIsSupported(message)) {
+        if (map.containsKey(message.getTag())){
+          map.get(message.getTag()).addLast(message);
+        } else {
+          // No such mapping with specified tag,
+          // Create first mapping with specified tag.
+          LinkedList<BlogMessage> firstValue = new LinkedList<BlogMessage>();
+          firstValue.addFirst(message); 
+          map.put(message.getTag(), firstValue);
+        }
+      } else {
+        System.err.println("Tag not supported. Tag: " + message.getTag());
       }
     }
-
-  }
-
-  public static int getSize(Map<String, LinkedList<BlogMessage>> map) {
-    return map.size();
   }
-
-  // Takes map, and
-  // Returns a linkedList of all values of BlogMessage type in the map. 
-  public static LinkedList<BlogMessage> getMessages(Map<String, LinkedList<BlogMessage>> map) {
-    System.out.println("Fetching all [LinkedList] Values in map...");
-
+  
+  /** FOR ALL getMessages() METHODS: 
```

Comment:
Consider using one style of comment for the same method, pick one of /**/ or //.

Reply:
Just to add to this, it seems that // comments are preferred over multi-line comments in Google code.